apache/ambari
# AMBARI-24230 - [Logsearch] Update hdfs configuration from Configurati…

…on Editor fails with 400 Error

## What changes were proposed in this pull request?

In the input configs, input config templates and the source code these names are not consistently spelled:
map_fieldname    map_field_name
new_fieldname    new_field_name
map_fieldvalue   map_field_value
map_fieldcopy    map_field_copy

Sometimes the first spelling is used and sometimes the second for the same object and the json parser can not parse the part of the input configs which are contains these names. It just omit their values.
Fix: Renamed all occurrences of these to the one in the 2nd column ones.

## How was this patch tested?

ambari-logsearch-it
mvn clean install -Dbackend-tests

Manually:
1. Login to Logsearch portal
2. Navigate to Configuration 
3. Select a service and update some configuration E.g Hdfs > update log file name
4. open chrome's inspect window and select "Network" tab
5. Click Save, configuration should be saved, but fails silently: check the entry in the inspect/network window.


Please review
@swagle @oleewere @miklosgergely @g-boros @adoroszlai
